ftwhttp/header.go

```diff
@@ -34,7 +35,9 @@ func (w stringWriter) WriteString(s string) (n int, err error) {
 
 // Add adds the key, value pair to the header.
 // It appends to any existing values associated with key.
+// The key is case in-sensitive
 func (h Header) Add(key, value string) {
```

Comment:
This method doesn't do what its comment says. I'll open a separate issue for that.

Reply:
https://github.com/coreruleset/go-ftw/issues/332